website/registrations/services.py

```diff
@@ -367,7 +368,7 @@ def process_entry_save(entry: Entry) -> None:
         # gets the welcome email
         emails.send_renewal_complete_message(entry.renewal)
 
-    entry.payment.paid_by = member
+    entry.payment.paid_by = PaymentUser.objects.get(pk=member.pk)
```

Comment:
Why does this need to be a PaymentUser exactly? The place where this property is used could convert it, then you don't need to import the model here

Reply:
So actually here we are doing a bit of a trick because we are 'fixing' the payment after the user has been created. So we are not using the native `create_payment` flow here.

Reply:
So yes, I did mean create payment. But that does not mean it is no longer true. The PaymentUser model is a proxy model. So it should not require you to get it before it saves it directly again right? The foreignkey only saves the pk and the proxy model doesn't change the database fields.

Reply:
Okay maybe it works, but it also feels weird to rely on the fact that its a proxy model. Better idea: use `create_payment` again. If a payment is already there, `create_payment` will update the payment so that would work.

Reply:
Nvm that fucks things up. I'll just go with Member and I'll add a comment explaining it